DungeonAlchemistImporter/DungeonAlchemist.js

```diff
@@ -293,7 +293,7 @@ const DungeonAlchemistImporter = (() => {
     if (mapGraphics.length == 0) {
       sendChat(
         "Dungeon Alchemist",
-        "You need to upload your map image and put it in the Map Layer before importing the line-of-sight data. Make sure that your map is in the background layer by right clicking on it, selecting \"Layer\" and choosing \"Map Layer\"."
+        "/w gm " + "You need to upload your map image and put it in the Map Layer before importing the line-of-sight data. Make sure that your map is in the background layer by right clicking on it, selecting \"Layer\" and choosing \"Map Layer\"."
```

Comment:
the "/w gm " should probably be in a global variable like whisperCommand or whisperPrefix.